### DISCUSSION TITLE
How open car2play

### DISCUSSION THREAD

**Discussion by italoc** [ludwig-v/wireless-carplay-dongle-reverse-engineering#127] [General]
Someone can explain me ho to open car2play device without broke a plastic case?

**Comment by 350d** [comment]
Check my photo for all clips exact locations:
![IMG_0165](https://user-images.githubusercontent.com/429208/227538259-401d842f-d459-4fa3-86fd-aeb79d82898e.JPG)
You can use some plastic card/tool to open all of them. Start from USB port side.
